.gitignore

```diff
@@ -45,4 +45,5 @@ fp-info-cache
 *.hex
 
 external
+!/vehicle/mkv/hardware/service_section/interface/external/
```

Comment:
Not sure why this is only an issue now. I think we should just have

```
/external
```

Or else maybe rename the board folder. We should probably reorganize all the folders. Idk whose bright idea the original organization was /s (it was me :disappointed:)